cromulent/model.py

```diff
@@ -913,6 +928,129 @@ def _toJSON(self, done, top=None):
 		else:
 			return d
 
+	def _toJSON_fast(self, done, top=None):
+		"""Serialize as JSON."""
+		# If we're already in the graph, return our URI only
+		# This should only be called from the factory!
+
+		# This should ONLY be used in Python3.6+
+		# as it relies on a stable order to a regular dict
+		# which was not the case before 3.6
+
+		# id, type, _label is the default.
+		if not self._factory.id_type_label and id(self) in done:
+			return self.id
+
+		# Can't pass in self as a param
+		if top is None:
+			top = self
+
+		# Add back context at the top, if set
+		result = {}
+		if top is self and not id(self) in done and self._factory.context_uri: 
```

Comment:
Stylistic comment: do you prefer `not id(self) in done` to `id(self) not in done`? I find the latter to be more readable.

Reply:
Agree. Fixed.